frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionView.tsx

```diff
@@ -27,6 +27,16 @@ const OngoingMissionSection = styled.div`
 const ButtonStyle = styled.div`
     display: block;
 `
+const OngoingMissionHeader = styled.div`
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+    justify-content: space-between;
+`
+const Square = styled.div`
```

Comment:
Rename to avoid shadowing build-in css "square"

E.g. StyledSquare

Reply:
This is actually not supposed to be in here, it's from the stop button code and I should have removed it from this commit. thank you Edda! I will remove this now